wfc3tools/calwf3.py

```diff
@@ -1,21 +1,11 @@
-from __future__ import print_function
-
 # STDLIB
 import os.path
 import subprocess
-from .version import __version_date__, __version__
 
 # STSCI
 from stsci.tools import parseinput
 from .util import error_code
 
-try:
-    from stsci.tools import teal
```

Comment:
Is WFC3 team ok with removing TEAL support? This seems out of scope.

Reply:
@FDauphin

Reply:
Back in September, I asked Michele De La Pena (might be helpful to ask her review as well) if we could remove TEAL support, but she said she was unsure. I am in support of removing it because I do not know in any instances where TEAL is the preferred method of using `wfc3tools`.

Reply:
I agree that TEAL has probably outlived its usefulness, but someone should do the due diligence to make sure all the doc you are deleting are already reflected in RTD docs. If not, they have to be ported. Otherwise, you might lose precious info. Up to you.

Reply:
@cshanahan1 is correct; the help files info is added as docstrings in PEP 8 style and is added in the RTD. However, the new RTD I initially removed all TEAL dependency. I suppose it would be best to add a sentence or two on the main RTD page that TEAL dependency was removed and why so.